ecosystem/indexer-server/doc/openapi.yaml

```diff
@@ -1,12 +1,12 @@
 openapi: 3.0.3
 info:
-  title: Aptos Dev API Specification
+  title: Aptos Indexer API Specification
   description: >
-    The Aptos Node API is a RESTful API for client applications to interact with the Aptos blockchain.
+    The Aptos Indexer API is a RESTful API for client applications to read Aptos Blockchain Data.
   license:
     name: Apache 2.0
     url: https://www.apache.org/licenses/LICENSE-2.0.html
-  version: 0.1.1
+  version: 1
```

Comment:
You should still use minor versions, I would stick with a non-major version until you're fully ready for release.  I'd make this 0.1.0.  Or st least 1.0.0

Reply:
no going to add version to the yaml for now